tests/Behat/DrupalContext.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_authentication\Behat;
+
+use Drupal\DrupalExtension\Context\DrupalContext as BaseDrupalContext;
+
+/**
+ * Defines step definitions specifically for testing the CAS users.
+ */
+class DrupalContext extends BaseDrupalContext {
```

Comment:
We already have an AuthenticationContext, why create a new one? Also, we should never extend from another Context. Only from RawContexts..